pkg/cmd/run.go

```diff
@@ -243,6 +243,10 @@ func (o *runCmdOptions) validate() error {
 		return err
 	}
 
+	if o.OutputFormat != "" && o.Dev {
+		return fmt.Errorf(`invalid dev specification "%t". Expected "--dev=false"`, o.Dev)
```

Comment:
I think it won't be clear for the user why this is invalid. I guess something like `cannot use --dev with -o/--output options` would be clearer.

Reply:
Thanks, I'll amend and force push